static/style.css

```diff
@@ -46,7 +46,7 @@ button {
     --background-light-gray: rgb(242, 242, 242);
```

Comment:
Ma perchè usiamo var globali in CSS quando abbiamo anche qui la prop globale `theme` di styled components?

Reply:
questo era già così quando io ho iniziato a sviluppare sul sito, quindi l'ho lasciato, ho semplicemente aggiunto il colore che mancava, si può pensare anche di rimuoverlo volendo, ma non so dirti per quale motivo all'inizio sia stato inserito